src/extensions/as-legacy-globals-for-extension-api/as-legacy-global-object-for-extension-api-with-modifications.ts

```diff
@@ -2,44 +2,15 @@
  * Copyright (c) OpenLens Authors. All rights reserved.
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
-import type { Injectable, TentativeTuple } from "@ogre-tools/injectable";
-import { getLegacyGlobalDiForExtensionApi } from "./legacy-global-di-for-extension-api";
-
-type MapInjectables<T> = {
-  [Key in keyof T]: T[Key] extends () => infer Res ? Res : never;
-};
+import { asLegacyGlobalForExtensionApi } from "./as-legacy-global-object-for-extension-api";
+import type { Injectable } from "@ogre-tools/injectable";
 
 export const asLegacyGlobalObjectForExtensionApiWithModifications = <
-  TInjectable extends Injectable<unknown, unknown, TInstantiationParameter>,
-  TInstantiationParameter,
-  OtherFields extends Record<string, () => any>,
+  InjectableInstance extends InjectionTokenInstance,
+  InjectionTokenInstance,
+  ModificationObject extends object,
```

Comment:
I would actually just remove this function since the `Object.assign` can easily be outside.

Reply:
True, we considered the same. Will do.

Reply:
Didn't do, as `Object.assign` exposes chance of doing it wrong.